Classes/Core/BaseTestCase.php

```diff
@@ -30,6 +30,9 @@ abstract class BaseTestCase extends TestCase
     /**
      * Creates a mock object which allows for calling protected methods and access of protected properties.
      *
+     * Note: This method has no native return types as having native (incomplete) return types, PHPDoc return type
+     * annotations and generics in combination tends to confuse PhpStorm's static type analysis.
```

Comment:
Mhh .. this sentens sounds kind weired. "has no native return typas as having native (inclomplete) return types..." and so go ...

Reply:
I have rephrased those two comments now. I hope that they now are less weird and less confusing. :-)